cmd/kops/update_cluster.go

```diff
@@ -133,6 +137,8 @@ func NewCmdUpdateCluster(f *util.Factory, out io.Writer) *cobra.Command {
 	viper.BindEnv("lifecycle-overrides", "KOPS_LIFECYCLE_OVERRIDES")
 	cmd.RegisterFlagCompletionFunc("lifecycle-overrides", completeLifecycleOverrides)
 
+	cmd.Flags().BoolVar(&options.DoDeferredDeletions, "cleanup-after-upgrade", options.DoDeferredDeletions, "Delete old revisions of cloud resources that were needed during an upgrade")
```

Comment:
Do we want to use `prune` instead?
```suggestion
	cmd.Flags().BoolVar(&options.DoDeferredDeletions, "prune", options.DoDeferredDeletions, "Delete old revisions of cloud resources that were needed during an upgrade")
```

Reply:
I like prune, let me make the change!